### DISCUSSION TITLE
active-monitor and orientation

### DISCUSSION THREAD

**Discussion by jorigel** [ltsp/ltsp#898] [Q&A]
Hi,
on ltsp 23.01 I've got multiple monitors partially of the same kind on different positions (left and right) on the clients. 

So I need to define the setup for a couple of clients individually e.g. the active-monitor and the orientation of it under lightdm and after login under the os.

What is the most straight way to solve this problem most likely via the ltsp.conf and does anybody has excamples how to do this?

thank You

**Comment by alkisg** [comment]
See the second example about custom xorg.conf there: https://ltsp.org/man/ltsp.conf/#examples

I.e. you'd create whatever xorg.conf you want, and you'd activate/symlink it under [mac] in ltsp.conf.

**Comment by jorigel** [comment]
Hi alkis,
setting up a xorg.conf for that is a real pain. Isn't there a possibility like in former times to run just a command like:

xrandr --output HDMI-0 --primary

through the ltsp.conf which would be much easier?

regards
jori

**Comment by alkisg** [comment]
The new LTSP doesn't have its own display manager, so you'd need to use the documentation of your DM (e.g. lightdm, gdm).
For example: https://askubuntu.com/questions/63681/how-can-i-make-xrandr-customization-permanent

**Comment by jorigel** [comment]
Hi alkis,
well, I dived deep into the facts of Your Trail. My display manager is lightdm.

The change in lstp.conf resulted in e.g.:

```
[pc-01]
LIGHTDM_CONF="display-setup-script=xrandr --output DVI-D-0 --primary --mode 1920
x1080 --pos 1920x611 --rotate normal --output HDMI-0 --mode 1920x1080 --pos 0x0 
--rotate normal --output DP-0 --off --output DP-1 --off
session-setup-script=xrandr --output DVI-D-0 --primary --mode 1920x1080 --pos 1920x750 --rotate normal --output HDMI-0 --mode 1920x1080 --pos 0x0 --rotate normal --output DP-0 --off --output DP-1 --off"
```
The log shows: lauching process 2711 xrandr . . . (upper order)
and: Process 2711 exited with return value 0

So I think it looks like it should have worked like I intended. But in fact neither the login-screen on DVI-D-0 worked (its on HDMI-0), nor the session screen worked (cause the HDMI-0 is rotated left and the DVI-D-0 has the wrong resolution on one of the clients for example)

There must be something after working the lightdm.conf that influences the tuning of the screens.
The _monitors.xml_ in the userland is allready deleted. So that can't disturb here.

Do You have any further idea what could intervene here? Never thought that this could be that complicated :)

thanX so far
regards jori
